src/NumericalAlgorithms/DiscontinuousGalerkin/Protocols.hpp

```diff
@@ -114,14 +114,16 @@ struct IsNumericalFluxCallable
  * Here's an example for a simple "central" numerical flux
  * \f$G(\boldsymbol{n}^\mathrm{int}, u^\mathrm{int},
  * \boldsymbol{n}^\mathrm{ext}, u^\mathrm{ext}) = \frac{1}{2}\left(
- * u_\mathrm{int} - n^\mathrm{int}_i n_\mathrm{ext}^i u_\mathrm{ext} \right)\f$
- * for the case where the interface normal is independent of the system
- * variables so
- * \f$\boldsymbol{n}_\mathrm{ext} = -\boldsymbol{n}_\mathrm{int}\f$
- * and therefore
- * \f$G=\frac{1}{2}\left(u_\mathrm{int} + u_\mathrm{ext} \right)\f$:
+ * n^\mathrm{int}_i F^i(u_\mathrm{int}) - n^\mathrm{ext}_i F^i(u_\mathrm{ext})
```

Comment:
Please use `F^{i,\mathrm{int}}_\alpha` instead of `F^i(u_\mathrm{int}})`

Reply:
I can't find a place in the code where `\alpha` is being used to index fluxes or variables and would prefer not to overload this PR with establishing a new notation. To make the sentence more clear I added a brief explanation of u and F in the fixup

Reply:
Please change the notation, thanks!